packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -253,7 +254,7 @@ export class CanvasManager implements CanvasManagerInterface {
   }
 
   private takeSnapshot(
-    isManualSnapshot: boolean,
+    enableManualSnapshot: boolean,
```

Comment:
Let's leave this as-is: I think it's subtle, but in the function we want to know if this current instance of `takeSnapshot` is a manual snapshot or not, but in the public interface, we are telling CanvasManager whether to enable or disable the manual snapshotting feature